org.salutatoi.Cagou.json

```diff
@@ -0,0 +1,1463 @@
+{
+    "app-id": "org.salutatoi.Cagou",
+    "default-branch": "0.7.0",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "sat_wrapper",
+    "finish-args": [
+        "--socket=session-bus",
+        "--socket=x11",
+        "--share=ipc",
+        "--socket=wayland",
+        "--share=network",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "python",
+            "config-opts": [
+                "--enable-deterministic-archives",
+                "--disable-static",
+                "--enable-shared",
+                "--with-ensurepip=yes",
+                "--with-system-expat",
+                "--with-system-ffi",
+                "--enable-loadable-sqlite-extensions",
+                "--with-dbmliborder=gdbm",
+                "--enable-unicode=ucs4"
+            ],
+            "post-install": [
+                "chmod 644 /app/lib/libpython2.7.so.1.0"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.python.org/ftp/python/2.7.16/Python-2.7.16.tar.xz",
+                    "sha256": "f222ef602647eecb6853681156d32de4450a2c39f4de93bd5b20235f2e660ed7"
+                }
+            ]
+        },
+        {
+            "name": "libxslt",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "ftp://xmlsoft.org/libxml2/libxslt-1.1.33.tar.gz",
+                    "sha256": "8e36605144409df979cab43d835002f63988f3dc94d5d3537c12796db90e38c8"
+                }
+            ]
+        },
+        {
+            "name": "setuptools",
```

Comment:
Are all these modules shared between Primitivus and Jp? I would consider keeping them in separate file and included as git submodule to ease maintenance.

Reply:
yes `Primitivus` and `jp` share most of the dependencies (there are some more for Primitivus, notably `urwid`). But maintenance is really easy on my side, I've made a script to generate the manifest with the right dependencies in the right order automatically, and it is using a template so I can tell the script to change anything specific. I can maintain things easily like this and using git submodules would in fact complicate my workflow.

Reply:
Alright, you're the captain here!